pipeline/transform.py

```diff
@@ -58,6 +58,13 @@ def clean_data(sales_data: dict) -> dict:
     }
 
 
+def clean_tags(tags: list[str]) -> list[str]:
+    """Removes leading/trailing whitespace and 'lowercases' a list of tags, returning it."""
+    if tags is None:
+        return []
+    return [tag.strip().lower() for tag in tags]
```

Comment:
nice normalisation. Some tags are proper nouns though. Should they be lowercase?

Reply:
Hmm...

Reply:
It's a trade off between presentability on the dashboard and minimising duplication of tags, and in this case, it's worth the small loss in dashboard presentability.